src/utilities.jl

```diff
@@ -331,3 +331,49 @@ function op_resolve_up_scopes(x, mn, scope, server)
 end
 
 maybe_lookup(x, server) = x isa SymbolServer.VarRef ? SymbolServer._lookup(x, getsymbolserver(server), true) : x # TODO: needs to go to SymbolServer
+
+function is_in_test_dir_of_package(path::String)
+    try # Safe failure - attempts to read disc.
+        if VERSION < v"1.1"
+            return false
+        else
```

Comment:
Maybe you just forgot to remove this check ?

Reply:
Yep!